README.md

```diff
@@ -169,7 +169,7 @@
 ## Other
 
 - [lsp.v](https://github.com/nedpals/lsp.v) - Implementation of the Language Server Protocol spec on V.
-- [setup-v-action](https://github.com/marketplace/actions/setup-v-environment) - Github action automation to use v in your workflow.
+- [setup-v-action](https://github.com/marketplace/actions/setup-v-environment) - GitHub action automation to use V in your workflow.
```

Comment:
Fix the typo as asked by @alexesprit in https://github.com/vlang/awesome-v/pull/57_